examples/openvino/aot_openvino_compiler.py

```diff
@@ -182,9 +182,17 @@ def main(
         if not dataset_path:
             msg = "Quantization requires a calibration dataset."
             raise ValueError(msg)
+
+        subset_size = 300
+        batch_size = calibration_dataset.batch_size
+        subset_size = (subset_size // batch_size) + int(subset_size % batch_size > 0)
+
+        transform_fn = lambda x: x[0]
```

Comment:
Lambda expressions are not allowed by coding standards of this project, so lintrunner checks fail. 
```suggestion
        def transform_fn(x):
            return x[0]
```